wsgi.py

```diff
@@ -0,0 +1,8 @@
+import os
```

Comment:
Is this running as a single process in the container? is it possible to use Gunicorn or awsgi? Or is it outside of this MVP?

Reply:
I've no problem with Gunicorn but I'm wondering if we should use uwsgi to be consequent with the other tutor services (for instance [edxapp](https://github.com/overhangio/tutor/blob/master/tutor/templates/build/openedx/Dockerfile#L195) )

Reply:
It was updated to use uwsgi in local and k8s.